src/bindings/PyDP/pydp_lib/casting.hpp

```diff
@@ -1,18 +1,17 @@
 // Convenient place to store type casters that are used through out project
 // This teaches pybind11 to cast types provided in the absl library
 
-
-#include "pybind11/stl.h"
 #include "absl/strings/string_view.h"
 #include "absl/types/optional.h"
+#include "pybind11/stl.h"
 
 namespace pybind11 {
-    namespace detail
-    {
-        template <>
-        struct type_caster<absl::string_view> : string_caster<absl::string_view, true> {};
+namespace detail {
+template <>
```

Comment:
can we add spacing here?
This makes the code unreadable

Reply:
Flattened namespaces can be seen also in the test files from C++ differential_privacy. I'm neutral to one format or the other.